presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayGreaterThanOrEqualOperator.java

```diff
@@ -49,18 +48,19 @@ private ArrayGreaterThanOrEqualOperator()
     @Override
     public FunctionInfo specialize(Map<String, Type> types, int arity, TypeManager typeManager, FunctionRegistry functionRegistry)
     {
-        Type type = types.get("T");
-        type = typeManager.getParameterizedType(StandardTypes.ARRAY, ImmutableList.of(type.getTypeSignature()), ImmutableList.of());
+        Type elementType = types.get("T");
+        Type type = typeManager.getParameterizedType(StandardTypes.ARRAY, ImmutableList.of(elementType.getTypeSignature()), ImmutableList.of());
         TypeSignature typeSignature = type.getTypeSignature();
-        return operatorInfo(GREATER_THAN_OR_EQUAL, RETURN_TYPE, ImmutableList.of(typeSignature, typeSignature), METHOD_HANDLE.bindTo(type), false, ImmutableList.of(false, false));
+        MethodHandle lessThanFunction = functionRegistry.resolveOperator(LESS_THAN, ImmutableList.of(elementType, elementType)).getMethodHandle();
```

Comment:
Oh right. You probably want to use both of them in the same loop then, so that it terminates early, if possible. I think to implement array1 <= array2, you want to use element1 > element2, and element1 < element2. Like:

```
for element1 in array1, element2 in array2:
  if element1 > element2:
    return false
 if element1 < element2:
   return true
# everything is equal
return true
```